ui/imports/shared/views/chat/CompactMessageView.qml

```diff
@@ -327,17 +329,22 @@ Item {
 
         UserImage {
             id: chatImage
+
             active: isMessage && headerRepeatCondition
+
             anchors.left: parent.left
             anchors.leftMargin: Style.current.padding
             anchors.top: chatReply.active ? chatReply.bottom :
                                             pinnedRectangleLoader.active ? pinnedRectangleLoader.bottom : parent.top
             anchors.topMargin: chatReply.active || pinnedRectangleLoader.active ? 4 : Style.current.smallPadding
+
             icon: root.senderIcon
             isIdenticon: root.isSenderIconIdenticon
-            onClickMessage: {
-                root.clickMessage(isProfileClick, isSticker, isImage, image, emojiOnly, hideEmojiPicker, isReply, false, "")
-            }
+            pubkey: senderId
+            name: senderDisplayName
+            showRing: !(root.amISender || Utils.getContactDetailsAsJson(senderId).isContact)
+
+            onClicked: root.clickMessage(true, false, false, null, false, false, false, false, "")
```

Comment:
Should this use the params like before?

Reply:
No, the signal emitted from `UserImage` before was hardcoded and did not bring any valuable information (it was: `root.clickMessage(true, false, false, null, false, false, isReplyImage)`, where `isReplyImage` was undefined..). Now `onClicked` signal does not propagate anything, just copy-pasted hardcoded parameters from old `UserImage` emit signal invocation here.